airbyte-integrations/connectors/source-xkcd/metadata.yaml

```
@@ -21,7 +25,7 @@ data:
   documentationUrl: https://docs.airbyte.com/integrations/sources/xkcd
   tags:
     - language:python
```

```suggestion
    - language:python
    - language:manifest-only
```

Only apply if you choose to delete everything except manifest+metadata.